spark/src/main/scala/org/apache/comet/CometSparkSessionExtensions.scala

```diff
@@ -378,6 +382,31 @@ class CometSparkSessionExtensions
             case None => b
           }
 
+        // For AQE shuffle stage on a Comet shuffle exchange
+        case s @ ShuffleQueryStageExec(_, _: CometShuffleExchangeExec, _) =>
+          val newOp = transform1(s)
+          newOp match {
+            case Some(nativeOp) =>
+              CometSinkPlaceHolder(nativeOp, s, s)
+            case None =>
+              s
+          }
+
+        // For AQE shuffle stage on a reused Comet shuffle exchange
+        // Note that we don't need to handle `ReusedExchangeExec` for non-AQE case, because
+        // the query plan won't be re-optimized/planned in non-AQE mode.
+        case s @ ShuffleQueryStageExec(
```

Comment:
curious if we can replace this rule with:
```scala
        case ReusedExchangeExec(_, op) => op
```
?

Reply:
Hmm, then users won't see `ReusedExchangeExec` anymore in explain string or Spark UI. It is useful to know which part is reused by Spark.

Reply:
oops you are right